shared/login/register/paper-key/index.js

```diff
@@ -34,5 +34,5 @@ PaperKey.propTypes = {
 }
 
 export default connect(
-  (state, ownProps) => ownProps.mapStateToProps(state)
+  state => ({waitingForResponse: state.login.waitingForResponse})
```

Comment:
i'm confused how this works. this isn't passed down to the render, so do we need it? same below